lib/devicetrust/authn/authn.go

```diff
@@ -111,3 +115,14 @@ func RunCeremony(ctx context.Context, devicesClient devicepb.DeviceTrustServiceC
 	}
 	return newCerts, nil
 }
+
+// RunCeremony performs the client-side device authentication ceremony.
+//
+// Device authentication requires a previously registered and enrolled device
+// (see the lib/devicetrust/enroll package).
+//
+// The outcome of the authentication ceremony is a pair of user certificates
+// augmented with device extensions.
+func RunCeremony(ctx context.Context, devicesClient devicepb.DeviceTrustServiceClient, certs *devicepb.UserCertificates) (*devicepb.UserCertificates, error) {
```

Comment:
Despite the comment above, it looks like this isn't used anymore, so we can also just drop it.